### PR TITLE
Styleguide | Replace Footer 

### PR DESCRIPTION
**Pull request by Sjones352**
Connects #3394 

AC:
* Use the [Footer](https://github.com/department-of-veterans-affairs/caseflow/blob/master/client/app/components/Footer.jsx) React component in `StyleGuideFooter.jsx`for the footer portion
* Use the [Navigation Bar](https://github.com/department-of-veterans-affairs/caseflow/blob/master/client/app/components/NavigationBar.jsx) React component for the Navigation bar portion
* Ensure tooltip for the word "built" still says "Recent build date goes here"

screen shot 2017-10-02 at 1 09 58 pm

### CODE REVIEW

File: client/app/containers/StyleGuide/StyleGuideFooter.jsx [hunk 1]
```diff
@@ -1,50 +1,27 @@
...
-      },
-      {
-        title: 'Sign out',
-        link: '#footer'
-      }
-    ];
+const options = [
+  {
+    title: 'Help',
+    link: '#navigation_bar'
```
Comment (mmlumba) [inline client/app/containers/StyleGuide/StyleGuideFooter.jsx]: Can you change this back to `#footer`

File: client/app/containers/StyleGuide/StyleGuideFooter.jsx [hunk 2]
```diff
@@ -99,25 +67,14 @@ export default class StyleGuideFooter extends React.Component {
...
-          </div>
-        </div>
-      </footer>
-    </div>
-  </div>;
+  <div className="sg-nav-wrap">
+   <Footer
+    appName="Hearing Prep"
+    buildDate="10/01/2017"
+    feedbackUrl="#" />
```
Comment (mmlumba) [inline client/app/containers/StyleGuide/StyleGuideFooter.jsx]: Change this to `#footer`

File: client/app/containers/StyleGuide/StyleGuideFooter.jsx [hunk 3]
```diff
@@ -99,25 +67,14 @@ export default class StyleGuideFooter extends React.Component {
...
-            </a>
-          </div>
-        </div>
-      </footer>
-    </div>
-  </div>;
+  <div className="sg-nav-wrap">
+   <Footer
+    appName="Hearing Prep"
+    buildDate="10/01/2017"
```
Comment (mmlumba) [inline client/app/containers/StyleGuide/StyleGuideFooter.jsx]: Its fine to leave this as a static variable here, but add a comment within `StyleGuideFooter` that states the recent build date is based off of "date" in `build_version.yml`

File: client/app/containers/StyleGuide/StyleGuideFooter.jsx (hunk 2 above)
Comment (Sjones352) [reply client/app/containers/StyleGuide/StyleGuideFooter.jsx]: Fixed

File: client/app/containers/StyleGuide/StyleGuideFooter.jsx (hunk 3 above)
Comment (Sjones352) [reply client/app/containers/StyleGuide/StyleGuideFooter.jsx]: Fixed

File: client/app/containers/StyleGuide/StyleGuideFooter.jsx (hunk 1 above)
Comment (Sjones352) [reply client/app/containers/StyleGuide/StyleGuideFooter.jsx]: Fixed

File: client/app/containers/StyleGuide/StyleGuideFooter.jsx [hunk 4]
```diff
@@ -60,29 +37,21 @@ export default class StyleGuideFooter extends React.Component {
     <p>
      Conveniently, if a developer hovers over the word
      “Built” they’ll see a tooltip showing the build date
-     of the app that they are viewing.</p>
-
+     of the app that they are viewing. In styleguide footer,  recent build date
```
Comment (nickheiner-usds) [inline client/app/containers/StyleGuide/StyleGuideFooter.jsx]: There's an extra space before 'recent'.

Comment (Sjones352) [reply client/app/containers/StyleGuide/StyleGuideFooter.jsx]: Fixed

**Pull request merged**